README.rst

```diff
@@ -1,6 +1,33 @@
 Cloudant-Python
 ===============
 
+This Version is Deprecated
+--------------------------
+
+As of 13 October 2015, this development branch is deprecated in favor of `the new 
+development branch starting at version 2.0.0a1 <https://github.com/cloudant/python-cloudant>`_. 
```

Comment:
Maybe you're right that I could put "repo" in the readme.md, which is automatically displayed on the GH repo.  But I think on the .rst file (here), which is displayed on PyPI, "branch" makes more sense because we're essentially taking over the 'cloudant' package..